### DISCUSSION TITLE
Building Instance segmentation on my data gave promising results but how can I improve them?

### DISCUSSION THREAD

**Discussion by Mikor-mkr** [opengeos/geoai#324] [General]
Hello everyone,

I have tried training a building instance segmentation model on my data but the results seem promising but with a lot of room for improvement. My raster image has 5cm GSD (Ground Sampling Distance) so its a very high-res image which I have split in 2 parts one for training and one for validation. The image is an orthophoto from a dense urban area.

<img width="1492" height="831" alt="image" src="https://github.com/user-attachments/assets/188abffb-f320-49e0-ac6b-45b2c746a884" />

I have tried training with the default parameters (as shown in the example of the documentation) and here are my results:

<img width="1498" height="833" alt="image" src="https://github.com/user-attachments/assets/5a0f4424-5f29-437f-9d98-8c177b191e23" />

I have also tried training with 25 epochs but the results where almost identical if not slightly worse. 

Any ideas on what to test in order to improve?

I would greatly appreciate any information that can help me understand in more depth how to improve this behavior and train more effectively.

**Comment by giswqs** [comment]
What do your training labels look like?

**Reply by Mikor-mkr** [reply]
Hello there,

here is an image of my training labels:
<img width="1496" height="837" alt="image" src="https://github.com/user-attachments/assets/799d0a67-e457-46dd-9c26-835150b95a8a" />

And here is an overview:
<img width="745" height="796" alt="image" src="https://github.com/user-attachments/assets/28d58236-6e78-409b-b10c-d0f33b06d422" />

There are 1826 polygons in total. The cut out area is the test image i am using.

I can also share the data if that's necessary.

**Comment by Mikor-mkr** [comment]
I have tried to do some data augmentation by rotating the images left and right and also made some attempts with learning rate scheduling but the results remain the same, if anyone has any idea I'm more than eager to listen and test it.
